charts/eraser/values.yaml

```diff
@@ -72,6 +72,7 @@ runtimeConfig:
         #     - HIGH
         #     - MEDIUM
         #     - LOW
+        #   ignoredStatuses:
```

Comment:
you can revert any changes to here. manifest_staging will automatically be promoted when we cut a new release

Reply:
Ah gotcha, just removed my changes here thanks!